src/bin/vip-dev-env-exec.js

```diff
@@ -38,6 +40,9 @@ command( { wildcardCommand: true } )
 	.argv( process.argv, async ( unmatchedArgs, opt ) => {
 		const slug = getEnvironmentName( opt );
 
+		const trackingInfo = getEnvTrackingInfo( slug );
+		await trackEvent( 'dev_env_exec_command_execute', trackingInfo );
```

Comment:
My concern here is that if `trackEvent()` fails for some reason, this will abort the whole thing.

// There is a bug in `trackEvent()`: because [this line](https://github.com/Automattic/vip/blob/develop/src/lib/tracker.js#L47) returns a `Promise` instead of `await`'ing it, the `catch` block won't catch any exceptions, and we will end up with an unhandled rejection

Reply:
I fixed the bug, now the catching should happen in trackEvent correctly.